### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ivandruzhinin**
* bump 'common' dependency to '3.44.0'
* bump 'common-utils' dependency to '0.0.3'
* make 'batchByGroup' setting default to 'false'

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -167,8 +167,8 @@ clean {
 dependencies {
     api platform('com.exactpro.th2:bom:3.2.0')
```
Comment (isengrims) [inline build.gradle:168]:
```suggestion
    api platform('com.exactpro.th2:bom:3.2.0')
    api platform('com.exactpro.th2:bom:4.1.0')
```

Comment (isengrims) [inline build.gradle:168]:
```suggestion
    api platform('com.exactpro.th2:bom:4.1.0')
```

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Outgoing message can be handled differently depending on send mode. There are 4
 + *appThreads* - amount of non-IO threads (session-count * 2 by default)
 + *maxBatchSize* - max size of outgoing message batch (`1000` by default)
 + *maxFlushTime* - max message batch flush time (`1000` by default)
-+ *batchByGroup* - batch messages by group instead of session alias and direction (`true` by default)
++ *batchByGroup* - batch messages by group instead of session alias and direction (`false` by default)
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:39]: Please hide this option from readme until book & page version

File: README.md [hunk 3]
```diff
@@ -126,6 +126,12 @@ spec:
...
 
+## 2.0.3
+
+* bump `common` dependency to `3.44.0`
+* bump `common-utils` dependency to `0.0.3`
+* make `batchByGroup` setting default to `false`
+
 ## 2.0.2
 
 * add channel flush timeout
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:137]: Please change `add support for session groups` to `prepared architecture for support session groups`

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**